templates/zerver/help/gdpr-compliance.md

```diff
@@ -112,7 +112,7 @@ GDPR:
   user's Zulip profile, which you can access from the right sidebar.
 * The [Zulip Cloud export](/help/export-your-organization) supports exporting
   all the data related to a Zulip user or organization.
-* The [Zulip REST API](https://zulipchat.com/api/rest) lets you
+* The [Zulip REST API](/api/rest) lets you
```

Comment:
Have you tested these links to see if they work? Or are they already tested?

Reply:
Yes, I tested them. Since both of them are in the `templates/zerver` directory this works fine.